### DISCUSSION TITLE
Syntax for duotone in transformOptions on Gatsby Image

### DISCUSSION THREAD

**Discussion by wad-im** [gatsbyjs/gatsby#31314] [Help]
Hello together, 

I feel stupid asking this, but can anyone help me point out the correct syntax on how to correctly pass in the object containing {highlight: string, shadow: string, opacity: number} to achieve a duotone effect in the GatsbyImage resolver or as a prop for static images. Somehow I fail to achieve this. 

Thank you,
Wadim

**Comment by vladar** [comment]
Check out this example as a reference:

https://github.com/gatsbyjs/gatsby/blob/e351588365c32ebf84c06048ba3b2da16e12793c/examples/using-gatsby-image/src/pages/index.js#L110-L120

**Reply by wad-im** [reply]
Thank you, Vladar. I tried this way many times. However, I am getting an error back that transformOptions is an  unknown argument on field "ContentfulAsset.gatsbyImageData". Is it because the option is not available if I load images from contentful.
